witness.schema.json

```diff
@@ -0,0 +1,393 @@
+  "$defs": {
+    "format-version": {
+      "description": "Version of the format.",
+      "const": "0.1"
+    },
+    "producer": {
+      "description": "Producer. Could be a tool, a human or something else.",
+      "type": "object",
+      "required": [
+        "name",
+        "version"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the producer."
+        },
+        "version": {
+          "type": "string",
+          "description": "Version of the producer."
+        },
+        "configuration": {
+          "type": "string",
+          "description": "Configuration of the producer. Should be used to distinguish different configurations of the same version of the producer."
+        },
+        "description": {
+          "type": "string",
+          "description": "Additional description about the producer. Could be used for any information that does not fit into any of the other producer properties."
+        },
+        "command_line": {
+          "type": "string",
+          "description": "Command line of the execution of the tool. Should be used to improve reproducibility."
+        }
+      }
+    },
+    "hash": {
+      "type": "string",
+      "description": "SHA-256 hash.",
+      "pattern": "[0-9a-fA-F]{64}"
+    },
+    "entry-uuid": {
+      "type": "string",
+      "description": "Unique identifier of the entry ([RFC4122](https://www.ietf.org/rfc/rfc4122.txt) defines the UUID format).",
+      "format": "uuid"
+    },
+    "task": {
+      "description": "Verification task.",
+      "type": "object",
+      "required": [
+        "input_files",
+        "input_file_hashes",
+        "specification",
+        "data_model",
+        "language"
+      ],
+      "properties": {
+        "input_files": {
+          "description": "Files belonging to the verification task that were given as input to the producer.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Bash-compliant file name pattern, which must represent _exactly one_ file."
+          }
+        },
+        "input_file_hashes": {
+          "type": "object",
+          "description": "SHA-256 hashes of each file in `input_files`. Every file name pattern listed in `input_files` must appear as a key.",
+          "additionalProperties": {
+            "$ref": "#/$defs/hash"
+          }
+        },
+        "specification": {
+          "type": "string",
+          "description": "Specification in [SV-COMP format](https://sv-comp.sosy-lab.org/2022/rules.php) against which the verification is done."
+        },
+        "data_model": {
+          "description": "Data model to be assumed for the task.",
+          "enum": [
+            "ILP32",
+            "LP64"
+          ]
+        },
+        "language": {
+          "description": "Source language of the input files.",
+          "enum": [
+            "C"
```

Comment:
This was not previously restricted to `C`, any source language was allowed.

Reply:
This came from the old schema, but I agree there's no reason to restrict it, especially since this format is also conceivable to be used in SV-COMP's Java track.